src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,24 +1,14 @@
 databaseChangeLog:
-  - changeSet:
-      id: 1
-      author: sa
-      changes:
-        - createSequence:
-            sequenceName: id_seq
-            incrementBy: 1
-            startValue: 1
             
   - changeSet:
-      id: 2
+      id: "create sequence and table for process type"
```

Comment:
Because you mentioned they don't have to be numeric in one of the comments. So I decided to give logical names. Also in one of the other comments you mentioned that we can use uuids if we are using numeric ids.